lib/node_modules/@stdlib/math/base/complex/divide/benchmark/benchmark_smith.js

```diff
@@ -0,0 +1,39 @@
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var randu = require( '@stdlib/math/base/random/randu' );
+var isArray = require( '@stdlib/assert/is-array' );
+var cdivSmith = require( './cdiv_smith.js' );
```

Comment:
You can alias as `cdiv`, as this file does not contain another `cdiv` implementation from which the Smith implementation must be differentiated.